opm/autodiff/FlowMainEbos.hpp

```diff
@@ -546,8 +543,7 @@ namespace Opm
             {
                 const EclipseGrid& inputGrid = eclState().getInputGrid();
                 eclipse_writer_.reset(new EclipseWriter(eclState(), UgGridHelpers::createEclipseGrid( grid , inputGrid )));
-                eclipse_writer_->writeInitial(geoprops_->simProps(grid),
-                                              geoprops_->nonCartesianConnections());
+                eclipse_writer_->writeInitial(computeLegacySimProps_(), eclState().getInputNNC());
```

Comment:
Note, this is not doing the same thing anymore as the grid might have added additional NNCs. These will now be gelected. `geoprops_->nonCartesianConnections()` might be a superset of `eclState().getInputNNC()`